pixart.js

```diff
@@ -1 +1,40 @@
 
+
+//1
+   $("button").click(function(event){ //gets button and sets function when clicked
+     $(".brush").css("background" , "pink") //gets brush and sets background color
+     event.preventDefault() //stops the blue default from showing
+   })
+
+//2
+$("input[type='text']").keyup(function(){
+  $(".brush").css("background" , "pink") //gets same background when enter is pushed in the input section
```

Comment:
This event is fired when any key is pressed (technically when released), therefore the background is set to pink when any key is pressed (in any input, with type of "text").